content/pt-br/_collections/navigations/footer.md

```diff
@@ -0,0 +1,55 @@
+---
+links:
+  - title: 'Sobre'
+    items:
+      - title: 'Fale conosco'
+        href: 'mailto:hello@nuxtjs.com'
+      - title: 'Enterprise support'
+        to: '/support'
+      - title: 'Empresa NuxtLabs'
+        href: 'https://nuxtlabs.com/'
+      - title: 'Software de código aberto'
+        href: 'https://github.com/nuxt'
+      - title: 'Parcerias'
+        to: '/partners'
+      - title: 'Telemetria'
+        href: 'https://github.com/nuxt/telemetry'
+  - title: 'Ecossistema'
+    items:
+      - title: 'Anúncios'
+        to: '/announcements'
+      - title: 'Contibua'
+        to: '/contribution-guide'
+      - title: 'Bate-papo com a gente'
+        href: 'https://discord.nuxtjs.org/'
+      - title: 'Eventos'
+        to: '/events'
+      - title: 'Patrocinadores'
+        to: '/sponsors'
+      - title: 'Times'
```

Comment:
`Equipas`?

Reply:
`Equipas` is from pt-pt. `Times` is practically the same thing as `Equipes`, I changed it to `Equipes` because its looks better.

Reply:
The `Equipas` is from pt-pt. `Times` is pretty much the same thing as `Equipes`, but when referring to `Development Team` it is usually `Time`, `Equipes` is not used much on this occasion.